.github/actions/compile-models/requirements.txt

```diff
@@ -630,11 +640,11 @@ regex==2024.11.6 \
     --hash=sha256:fdd6028445d2460f33136c55eeb1f601ab06d74cb3347132e1c24250187500d9 \
```

Comment:
pyreadline3 is removed here https://github.com/openvinotoolkit/npu_compiler/pull/76/files#diff-4ce361c39542bd48a9bf135d5ca98ec3f07221a7497e8f1bb79494b46d112763L475